lib/matplotlib/colors.py

```diff
@@ -1858,9 +1858,34 @@ def linear_width(self, value):
 
 
 class PowerNorm(Normalize):
-    """
+    r"""
     Linearly map a given value to the 0-1 range and then apply
     a power-law normalization over that range.
+
+    Parameters
+    ----------
+    gamma : float
+        Power law exponent.
+    vmin, vmax : float or None
+        If *vmin* and/or *vmax* is not given, they are initialized from the
+        minimum and maximum value, respectively, of the first input
+        processed; i.e., ``__call__(A)`` calls ``autoscale_None(A)``.
+    clip : bool, default: False
+        If ``True`` values falling outside the range ``[vmin, vmax]``,
+        are mapped to 0 or 1, whichever is closer, and masked values
+        remain masked.
+
+        Clipping silently defeats the purpose of setting the over, under,
+        and masked colors in a colormap, so it is likely to lead to
```

Comment:
why is the masked color's purpose defeated?

Also, if it is kept, would it be more consistent to call it the "bad color" (as the getter/setter is `colormap.[gs]et_bad()`. (I'm a bit torn on this myself, as I think "bad" is a bit of a hard concept to get across, but it also covers NaN values as well as ma.masked values, which "masked" would seem to indicate exclusively)

Reply:
👍  - this was copied from the `Normalize` docstring, so I've fixed it there (and in the `LogNorm` docstring) too, and double checked the logic to confirm that this should indeed not comment on bad/masked values.